image_provisioner/playbooks/roles/seal-image/tasks/main.yaml

```diff
@@ -1,7 +1,8 @@
 ---
-- name: clean yum cache
+- name: clean yum cache final time before bundle ami
```

Comment:
nit: Seal image is used for qcow2 also, not just AMIs.   Comment a little misleading.